proctor-common/src/test/java/com/indeed/proctor/common/ProctorResultTest.java

```diff
@@ -75,7 +78,9 @@ public void testUnmodifiableView() {
         allocations.put("allocation1", new Allocation());
         final Map<String, ConsumableTestDefinition> definitions = new TreeMap<>();
         definitions.put("test1", new ConsumableTestDefinition());
-        final ProctorResult proctorResult1 = new ProctorResult("", buckets, allocations, definitions);
+        final Map<String, Object> inputContext = new HashMap<>();
```

Comment:
in line 69~70 there is equality check for empty identifiers and empty input context.
Should we do the same for non-empty ones?